docker/Dockerfile

```diff
@@ -68,11 +55,17 @@ ARG FQL_VER
 RUN go build -o /fql -ldflags="-X 'github.com/janderland/fql/internal/app.Version=${FQL_VER}'"
 
 
-# The final stage builds the 'fql' image.
+# The final stage builds the minimal 'fql' runtime image.
 FROM debian:12
 
-COPY --from=gobuild /fdb.deb /fdb.deb
-RUN dpkg -i ./fdb.deb &&\
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y curl=7.88.* && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+ARG FDB_LIB_URL
+RUN curl -fsSL ${FDB_LIB_URL} -o /fdb.deb && \
```

Comment:
Use `FENV_FDB_VER` here to generate the FDB URL. In fact, use `FENV_FDB_VER` in place of `FDB_VER` everywhere in this repo.